spec/actions/create_next_snapshot_version_action_spec.rb

```diff
@@ -35,6 +36,28 @@
         files_to_update_without_prerelease_modifiers: ['./test_file4.swift', './test_file5.kt']
       )
     end
+
+    it 'skips if current version is SNAPSHOT' do
+      expect(Fastlane::Helper::RevenuecatInternalHelper).not_to receive(:validate_local_config_status_for_bump)
+        .with("bump/#{next_version}", github_pr_token)
+      expect(Fastlane::Helper::RevenuecatInternalHelper).not_to receive(:calculate_next_snapshot_version)
+        .with(current_version_snapshot)
+      expect(Fastlane::Helper::RevenuecatInternalHelper).not_to receive(:create_new_branch_and_checkout)
+        .with(new_branch_name)
+      expect(Fastlane::Helper::RevenuecatInternalHelper).not_to receive(:replace_version_number)
+        .with(current_version_snapshot, next_version, ['./test_file.sh', './test_file2.rb'], ['./test_file4.swift', './test_file5.kt'])
+      expect(Fastlane::Helper::RevenuecatInternalHelper).not_to receive(:commmit_changes_and_push_current_branch)
+        .with('Preparing for next version')
+      expect(Fastlane::Helper::RevenuecatInternalHelper).not_to receive(:create_pr_to_main)
+        .with('Prepare next version: 1.13.0-SNAPSHOT', nil, repo_name, new_branch_name, github_pr_token, labels)
```

Comment:
For all these, since we are expecting `not_to` receive these calls, we don't need to specify the parameters:
```suggestion
      expect(Fastlane::Helper::RevenuecatInternalHelper).not_to receive(:validate_local_config_status_for_bump)
      expect(Fastlane::Helper::RevenuecatInternalHelper).not_to receive(:calculate_next_snapshot_version)
      expect(Fastlane::Helper::RevenuecatInternalHelper).not_to receive(:create_new_branch_and_checkout)
      expect(Fastlane::Helper::RevenuecatInternalHelper).not_to receive(:replace_version_number)
      expect(Fastlane::Helper::RevenuecatInternalHelper).not_to receive(:commmit_changes_and_push_current_branch)
      expect(Fastlane::Helper::RevenuecatInternalHelper).not_to receive(:create_pr_to_main)
```